packages/model-viewer/package.json

```diff
@@ -45,6 +45,7 @@
   "typings": "lib/model-viewer.d.ts",
   "types": "lib/model-viewer.d.ts",
   "scripts": {
+    "postinstall": "node ../../scripts/postinstall.js",
```

Comment:
That's pretty magical, thanks! Should we add this into our `update:package-lock` script? That's what I run anytime I update our versions.

Reply:
Checked it right now, it already runs since the update:package-lock runs npm install in every sub project.

Reply:
Ah, great; I didn't realize `postinstall` was a standardized npm command. Thanks!